ko/sql/query/show.rst

```diff
@@ -1359,13 +1353,13 @@ SHOW INDEX HEADER
 
 ::
 
-    SHOW INDEX HEADER OF table_name.index_name;
+    SHOW INDEX HEADER OF [ schema_name. ] table_name.index_name;
```

Comment:
It is advisable to remove all spaces in the form that contains spaces in the middle, such as `[ schema_name. ] table_name`.
Or it should be expressed as below.
  `schema_name.table_name | table_name`

Reply:
Thanks, I'll change it.
```
[schema_name.]table_name
```